<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="addRecord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QLabel" name="deleteRecord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QToolButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>790</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ОК</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="bdWindow">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>1001</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameMaterial">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>название материала</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="squarePore">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>площадь поры</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviationSquare">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>откл. от площ.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="porisity">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>530</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>пористость</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviationPorisity">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>530</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>откл. от порист.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numRecord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>630</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>номер записи</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
